docs/developer/writing-exporter-flow-components.md

```diff
@@ -0,0 +1,55 @@
+# Create Prometheus Exporter Flow Components
+
+This guide will walk you through the process of creating a new Prometheus exporter Flow component and best practices for implementing it. It is required that the exporter has an existing integration in order to wrap it as a Flow component.
+
+## Arguments (Configuration)
+
+`Arguments` struct defines the arguments that can be passed to the component. In most cases, this would be exactly the same as the arguments that the integration for this exporter the uses. Some recommendations:
```

Comment:
The full sentence is: "Avoid creating components with `Singleton: true` as it will make it impossible to run multiple instances of the exporter." would be that enough?